docs/install_linux.md

```diff
@@ -43,12 +43,29 @@ sudo dnf config-manager --add-repo https://cli.github.com/packages/rpm/gh-cli.re
 sudo dnf install gh --repo gh-cli
 ```
 
+<details>
+<summary>Show dnf5 commands</summary>
+
+If you're using `dnf5`, commands will vary slightly:
+
+```bash
+sudo dnf install dnf5-plugins
+sudo dnf config-manager addrepo --from-repofile=https://cli.github.com/packages/rpm/gh-cli.repo
+sudo dnf install gh --repo gh-cli
```

Comment:
Would it make more sense to be specific about the invocation here? It seems like still possible to have `dnf` refer to `dnf4` if you just [install](https://github.com/rpm-software-management/dnf5?tab=readme-ov-file#installing-dnf5) with `sudo dnf install dnf5`.

I'm open to either option, since I imagine most using `dnf5` have installed with the option to obsolete `dnf4`. But there might be no harm in being more specific 😁 

Let me know what you think @its-miroma ❤️ 

```suggestion
sudo dnf5 install dnf5-plugins
sudo dnf5 config-manager addrepo --from-repofile=https://cli.github.com/packages/rpm/gh-cli.repo
sudo dnf5 install gh --repo gh-cli
```

Reply:
It's a good temporary solution until `dnf5` is the default.